build/build.msbuild

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?> 
 <Project ToolsVersion="12.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
    <Import Project="$(MSBuildThisFileDirectory)\Build.Settings" /> 
+
+
    <ItemGroup> 
      <Projects Include="$(SolutionDir)\src\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj" /> 
      <Projects Include="$(SolutionDir)\src\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" /> 
      <Projects Include="$(SolutionDir)\src\AspNetCoreModuleV2\RequestHandler\RequestHandler.vcxproj" /> 
      <Projects Include="$(SolutionDir)\test\CommonLibTests\CommonLibTests.vcxproj">
    </ItemGroup> 
  
-   <Target Name="Build"> 
+   <Target Name="Build">
```

Comment:
Right now it doesn't shutdown IIS. All it does is if we have admin permissions, update the schema with what is installed in this repository. I wanted this change because we wouldn't need to update our CI agents to have the new schema.

Reply:
What would happen if the build is started on linux?
Is this script used for anything outside this repo (can we turn it into a target)?
Does it fail the build if the schema is not able to install with a correct message?

Reply:
I'll remove running this script in Build until we get a schema change.